src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentService.kt

```diff
@@ -250,6 +254,7 @@ class CodyAgentService(private val project: Project) : Disposable {
               try {
                 callback.accept(agent)
               } catch (e: Exception) {
+                logger.warn(e)
```

Comment:
The error: `IllegalStateException("Cannot start new session when one is already active")` was there but we ignored it 🙃